src/Integration/Binding/BindingController.cs

```diff
@@ -92,59 +91,44 @@ protected override int OnQueryStatus(ref Guid pguidCmdGroup, uint cCmds, OLECMD[
             return base.OnQueryStatus(ref pguidCmdGroup, cCmds, prgCmds, pCmdText);
         }
 
-        private bool OnBindStatus(ProjectViewModel projectVM)
```

Comment:
Previously there were two versions of _OnBind_ and _OnBindStatus_ taking different inputs. Only one is required now.